### PR TITLE
Add initial version of the Lagom 1.4 and Kubernetes orchestration blog post

### PR DESCRIPTION
**Pull request by yannickdeturck**
Todos before merging:
- ~Process feedback reviews~
- ~Demand feedback of Lightbend~
- ~Process feedback of Lightbend~
- ~Await final "go" of Lightbend~
- ~Update file name to set publication date~
- Publish the post

### CODE REVIEW

File: _posts/2018-05-11-lagom-1-4-and-kubernetes-orchestration.md [hunk 1]
```diff
@@ -0,0 +1,643 @@
...
+5. [Building and publishing Docker images](#building-and-publishing-docker-images)
+6. [Locally deploying to Kubernetes using Minikube](#locally-deploying-to-kubernetes-using-minikube)
+7. [Generating Kubernetes resources for production](#generating-kubernetes-resources-for-production)
+8. [Conclusion](#conclusion)
+9. [Extra resources](#extra-resources)
+
+## Introduction
+In this blog post we will take a closer look at the [Lightbend Orchestration tools](https://developer.lightbend.com/docs/lightbend-orchestration-kubernetes/latest/index.html){:target="_blank" rel="noopener noreferrer"}.
+Tools helping you deploy your Lagom application to Kubernetes and DC/OS.
+It was already possible to deploy Lagom application to Kubernetes as [this guide](https://developer.lightbend.com/guides/lagom-kubernetes-k8s-deploy-microservices/){:target="_blank" rel="noopener noreferrer"} demonstrates but it involved more manual tasks and having written the Kubernetes resource and configuration files yourself, as it usually goes.
```
Comment (tomvdbulck) [inline _posts/2018-05-11-lagom-1-4-and-kubernetes-orchestration.md]: as ... demonstrates, but it involved more manual tasks and having to write the Kubernetes ...

File: _posts/2018-05-11-lagom-1-4-and-kubernetes-orchestration.md [hunk 2]
```diff
@@ -0,0 +1,643 @@
...
+      scalaTest,
+      "org.webjars" % "foundation" % "6.2.3",
+      "org.webjars" % "foundation-icon-fonts" % "d596a3cfb3"
+    ),
+    EclipseKeys.preTasks := Seq(compile in Compile),
+    httpIngressPaths := Seq("/")
+  )
+```
+
+If you also have a frontend module it is important to define the `httpIngressPaths`, as you might have seen in the code sample above, in order to have your frontend accessible from outside the cluster.
```
Comment (tomvdbulck) [inline _posts/2018-05-11-lagom-1-4-and-kubernetes-orchestration.md]: in order to have your frontend be accessible

File: _posts/2018-05-11-lagom-1-4-and-kubernetes-orchestration.md [hunk 3]
```diff
@@ -0,0 +1,643 @@
...
+$ sbt
+```
+
+And deploy everything to Minikube using a sbt task that comes with the sbt-reactive-app plugin.
+The task however is not yet supported on Windows unfortunately.
+```sbtshell
+deploy minikube
+```
+
+During the setup we encountered an connection initialisation error of Helm:
```
Comment (tomvdbulck) [inline _posts/2018-05-11-lagom-1-4-and-kubernetes-orchestration.md]: we encountered a connection

File: _posts/2018-05-11-lagom-1-4-and-kubernetes-orchestration.md [hunk 4]
```diff
@@ -0,0 +1,654 @@
...
+order-v1-0-snapshot-5884754595-65wxd     1/1       Running   0          4m
+order-v1-0-snapshot-5884754595-wdbng     1/1       Running   0          4m
+reactive-sandbox-74fd955ddd-cjpw8        1/1       Running   7          6d
+```
+
+## Conclusion
+Upgrading to Lagom 1.4 and Play 2.6 went pretty smooth as the migration guides of Lightbend cover mostly everything in detail as usual.
+
+The orchestration tools make it pretty easy to test your Lagom application running in Kubernetes locally while still having the possibility to fine-tune the generated resource and configuration files.
+Integrating the tool suite into our project went smooth.
```
Comment (andreasevers) [inline _posts/2018-05-11-lagom-1-4-and-kubernetes-orchestration.md]: went smoothly

File: _posts/2018-05-11-lagom-1-4-and-kubernetes-orchestration.md [hunk 5]
```diff
@@ -0,0 +1,654 @@
...
+Kubernetes has gained a lot of popularity lately and with this, Lagom shows that it wants to embrace kubernetes to deploy your applications onto next to ConductR.
```
Comment (andreasevers) [inline _posts/2018-05-11-lagom-1-4-and-kubernetes-orchestration.md]: Kubernetes instead of lower case kubernetes

**Pull request merged**